### PR TITLE
Improved home layout, still not finished

### PR DESCRIPTION
**Pull request by rinoma**
Relates #37

### CODE REVIEW

File: src/controllers/home.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+const {findLatest} = require('../models/queries');
+
 module.exports = (req, res) => {
-  res.render('home');
+  findLatest()
+  .then(latest => {console.log(latest);res.render('home', {latest})})
```
Comment (Karyum) [inline src/controllers/home.js:5]: does this actually work?

**Review by Karyum — COMMENTED** [review]
:+1:

**Review by tyeung2017 — APPROVED** [review]
good job bro, mario seldom gives thumbs up

File: src/controllers/home.js (hunk 1 above)
Comment (rinoma) [reply src/controllers/home.js:5]: yes 100%
working

**Pull request merged**